protected/models/PatientDiagnosisParameter.php

```diff
@@ -129,7 +136,7 @@ public function query($searchProvider)
                 // Do nothing extra.
                 break;
             case 'NOT LIKE':
-                $query = "SELECT DISTINCT p1.id 
+                $query = "SELECT DISTINCT p1.id
```

Comment:
I know it would be horrible, when editing old modules, but it would be nice if we could have "auto-right trim" enabled in phpstorm...